ocs-framework/src/main/scala/ocs/framework/dsl/CswServices.scala

```diff
@@ -39,6 +41,8 @@ class CswServices(
     romaineFactory.redisAsyncApi(locationService.redisUrI(masterId))
   }
 
+  val scheduler: TimeServiceScheduler = this.timeServiceScheduler
```

Comment:
You should probably rename `timeServiceScheduler`  to just `scheduler` at line 31 and then you wont need this re-assignment.

Reply:
Sounds good.  Change made.